cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectorIT.java

```diff
@@ -152,7 +152,7 @@ public void testConnectorNotUpdatedWhenConfigUnchanged(VertxTestContext context)
         );
 
         KafkaConnectAssemblyOperator operator = new KafkaConnectAssemblyOperator(vertx, pfa, ros,
-                ClusterOperatorConfig.fromMap(Collections.emptyMap(), KafkaVersionTestUtils.getKafkaVersionLookup()),
+                ClusterOperatorConfig.buildFromMap(Collections.emptyMap(), KafkaVersionTestUtils.getKafkaVersionLookup()),
```

Comment:
I think we tend to use `Map.of` in these places instead of `Collections.emptyMap()` => at least in the newer code. So maybe you can use it in this PR as well (not just here but also below etc.).